scripts/refresh-after-snapshot.py

```
@@ -0,0 +1,620 @@
+
+from __future__ import annotations
+
+import json
+import os
+import subprocess
+import sys
+import time
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Callable
```

Ruff (0.15.17)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)